api/templates/api.mustache

```diff
@@ -359,10 +388,10 @@ func (a *{{{classname}}}Service) {{nickname}}Execute(r {{#structPrefix}}{{&class
 			var v {{{dataType}}}
 			err = a.client.decode(&v, localVarBody, localVarHTTPResponse.Header.Get("Content-Type"))
 			if err != nil {
-				newErr.error = err.Error()
+				newErr.error = _fmt.Sprintf("%s%v", errorPrefix, err.Error())
 				return {{#returnType}}localVarReturnValue, {{/returnType}}newErr
 			}
-			newErr.model = &v
```

Comment:
I think we need to leave this line as well, some commands rely on access to `model` to conditionally switch on error code.

Reply:
Cool, good to know that is needed, I put it back in and everything still seems to work the same way for these error messages.